scripts/30mini-iso-menu

```diff
@@ -0,0 +1,112 @@
+#!/bin/sh
+
+PREREQ=""
+
+prereqs()
+{
+       echo "$PREREQ"
+}
+
+case $1 in
+# get pre-requisites
+prereqs)
+       prereqs
+       exit 0
+       ;;
+esac
+
+. /scripts/casper-functions
+. /scripts/casper-helpers
+
+mountpoint=/cdrom
+
+iso_chooser_step1() {
+    # download JSON of simplestreams for showing the list of ISOs we might
+    # chain-boot to, hand that off to the menu, look what the choice was,
+    # use the requested ISO size to figure out if we have the memory or not,
+    # and kexec to reserve that memory
+
+    echo "###########################################"
+    echo "iso_chooser_step1"
+    echo "###########################################"
+
+    /usr/sbin/agetty --skip-login \
+        --login-program /usr/bin/iso-menu-session \
+        tty1 linux
```

Comment:
This whole area of things makes my head hurt tremendously but I'm not sure we want tty1 here? Because in at least some cases we'll want it to run on the serial console. do_interactive_netmount just reads and writes from stdin/stdout/stderr -- do we even need agetty?

Reply:
I used agetty as part of getting a terminal that ncurses was happy with.  (I wonder if casper-getty would suffice?)
The rest, I don't know yet.

Reply:
I think you might be able to get away with passing - instead of tty1? Not sure, sorry.

Reply:
That crashes agetty, actually.  The manpage documents some assumptions, perhaps those are not valid assumptions in this state.
Not implemented.